pom.xml

```diff
@@ -100,16 +100,28 @@
         </exclusion>
       </exclusions>
     </dependency>
-
     <!-- Needed for audit functionality -->
-    <!--
     <dependency>
```

Comment:
The readme says that audit isn't included by default -- should this be commented out then?  Or is there some dependency that needs it?